we_count/README.md

```diff
@@ -0,0 +1,22 @@
+# we_count
+
+> We Count website
+
+## Build Setup
+
+``` bash
+# install dependencies
+$ npm run install
+
+# serve with hot reload at localhost:3000
+$ npm run dev
+
+# build for production and launch server
+$ npm run build
+$ npm run start
```

Comment:
what is the difference between the `build` and `start` script. It would be good to articulate what to use which.

Reply:
There same unless you specify specific task to complete during build. In that case `npm run build` will also complete these tasks when the script runs. I have just been using `npm run dev` so far